src/Client/ConfigWindow.java

```diff
@@ -802,6 +801,28 @@ public void actionPerformed(ActionEvent e) {
     generalPanelCameraMovableRelativeCheckbox.setToolTipText(
         "Camera movement will follow the player position");
 
+    addSettingsHeader(generalPanel, "Menu/Item patching");
+
+    generalPanelBypassAttackCheckbox = addCheckbox("Always left click to attack", generalPanel);
+    generalPanelBypassAttackCheckbox.setToolTipText(
+        "Left click attack monsters regardless of level difference");
+    generalPanelBypassAttackCheckbox.setBorder(new EmptyBorder(7, 0, 10, 0));
+
+    generalPanelCommandPatchEdibleRaresCheckbox =
+        addCheckbox("Disable the ability to ingest holiday items or rares", generalPanel);
+    generalPanelCommandPatchEdibleRaresCheckbox.setToolTipText(
+        "Applies to the Easter Egg, Pumpkin, and the Half Wine.");
+
+    generalPanelCommandPatchQuestCheckbox =
+        addCheckbox("Swap eat & use options on Quest Items", generalPanel);
+    generalPanelCommandPatchQuestCheckbox.setToolTipText(
+        "Applies to giant Carp, chocolaty milk, Rock cake, and nightshade.");
```

Comment:
was thinking maybe we need to have replaced chocolaty milk -> whisky hero's quest (item id 584)

Reply:
Chocolaty milk should no longer have eat/use swapped but whisky should?

Reply:
the one of the hero's quest is a bit of trouble to get (some people may want to avoid miss drinking that one). Chocolaty milk can be regularly made without issue any time

Reply:
the one of the hero's quest is a bit of trouble to get (some people may want to avoid miss drinking that one). Chocolaty milk can be regularly made without issue any time